eng/scripts/Language-Settings.ps1

```diff
@@ -308,11 +308,11 @@ function PackageDependenciesResolve($artifactNamePrefix, $packageDirectory) {
   $artifactDownloadOutput = mvn `
     dependency:copy `
     -Dartifact="$pomArtifactName" `
-    -DoutputDirectory="$packageDirectory"
+    -DoutputDirectory="$packageDirectory" | Out-Null
```

Comment:
What is the point of both sending to Out-Null and capturing it in artifiactDownloadOutput?

Reply:
We noticed that the `java - docindex` pipeline was failing so this was part of an approach to silence errors coming from `mvn` so that a failure of `mvn` wouldn't cause the script to fail. 

The correct fix is here: https://github.com/Azure/azure-sdk-tools/pull/1977